rust/src/io/object_store.rs

```diff
@@ -49,9 +49,34 @@ impl std::fmt::Display for ObjectStore {
     }
 }
 
+/// BUild S3 ObjectStore using default credential chain.
+async fn build_s3_object_store(uri: &str) -> Result<Arc<dyn OSObjectStore>> {
+    use aws_config::meta::region::RegionProviderChain;
+    use aws_credential_types::provider::ProvideCredentials;
+
+    let region_provider = RegionProviderChain::default_provider().or_else("us-west-1");
```

Comment:
how come this is us-west-1 but it's us-west-2 down below on 71?

Reply:
A typo. Fixed.